README.md

```diff
@@ -1,14 +1,37 @@
 # Auth Info Dumper
 
-This is a PS4 payload to dump [auth info](https://www.psdevwiki.com/ps4/Auth_Info) from a SELF file. Right now it is hard-coded file name to dump and it is only work on 9.0.
+This is a PS4 payload to dump [auth info](https://www.psdevwiki.com/ps4/Auth_Info) from a ELF, SELF, or eboot.bin file. This payload only works on firmware 9.00!
+
+## How to use
+
+1. Download .bin from [Github Actions](test) or build it yourself!
```

Comment:
Better to remove the Actions link until it is actually available.

Reply:
DID I NOT REMOVE THAT-